ambry-frontend/src/main/java/com/github/ambry/frontend/s3/S3PutHandler.java

```diff
@@ -88,6 +88,9 @@ protected void doHandle(RestRequest restRequest, RestResponseChannel restRespons
     }
     restRequest.setArg(Headers.AMBRY_CONTENT_ENCODING, restRequest.getArgs().get(Headers.CONTENT_ENCODING));
 
+    // Overwrites are allowed by default for S3 APIs https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html
+    restRequest.setArg(NAMED_UPSERT, true);
```

Comment:
QQ: should we also enable this for multiupload?

Reply:
right, seems like we need to do this for all uploads.

Reply:
Sure. Thank you for mentioning it! Updated the code